features/fields_extensie.feature

```diff
@@ -1,126 +1,570 @@
+      | datum.datum (01.03.10)                    | 19860401 |
+      | land.code                                 | 6030     |
+      | plaats.code                               | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                             |
+      | type                | RaadpleegMetBurgerservicenummer                    |
+      | burgerservicenummer | 999994086                                          |
+      | fields              | geboorte.plaats.code                               |
+      Dan heeft de persoon met burgerservicenummer '999994086' de volgende 'geboorte' gegevens
+      | naam                 | waarde       |
+      | geboorte.plaats.code | 0518         |
+      En heeft de persoon GEEN andere properties
+
+    Scenario: opvragen veld met fields door opgeven laatste deel van het pad naar het veld
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                      |
+      | voornamen                 | Johannnes Geurt |
+      | geslachtsnaam             | Janssen         |
+      | aanduidingNaamgebruik     | E               |
+      En de persoon heeft de volgende 'geboorte' gegevens
+      | naam                                      | waarde   |
+      | datum (01.03.10)                          | 19860401 |
+      | land                                      | 6030     |
+      | plaats                                    | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                             |
+      | type                | RaadpleegMetBurgerservicenummer                    |
+      | burgerservicenummer | 999994086                                          |
+      | fields              | plaats.code |
+      Dan heeft de persoon met burgerservicenummer '999994086' de volgende 'geboorte' gegevens
+      | naam        | waarde       |
+      | plaats.code | 0518         |
+      En heeft de response GEEN andere properties
+
+  Rule: De response bevat alleen de properties die in de fields parameter gevraagd.
+    Scenario: Slechts één enkel attribuut wordt gevraagd
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999994086                       |
+      | fields              | geslachtsaanduiding             |
```

Comment:
Wat wordt hier precies beschreven/gevalideerd? geslachtaanduiding is een type (waardetabel) dus zou je met fields=geslachtsaanduiding krijgt je alle properties van geslachtsaanduiding terug, maar in de dan stap wordt alleen de code waarde gevalideerd